libcruby_sys.gemspec

```diff
@@ -0,0 +1,20 @@
+Gem::Specification.new do |s|
+  s.name        = 'libcruby_sys'
+  s.version     = '0.1.0'
+  s.licenses    = ['MIT']
```

Comment:
unknown. does it complain if you take it out for now?

Reply:
To be clear, it will definitely be _some_ permissive oss license – just not sure which one(s) yet. Didn't want to leave this in and forgot about it.

Reply:
Should we use whatever is in the existing libcruby-day?

Reply:
Since we are doing a clean room implementation, we basically get a free chance to relicense so we should use the opportunity to reevaluate